apps/user/repositories/user.repository.ts

```diff
@@ -254,7 +254,7 @@ export class UserRepository {
   });
   }
 
-  async findUserForPublicProfile(queryOptions: UserQueryOptions): Promise<UsersProfile> {
+  async findUserForPublicProfile(queryOptions: UserQueryOptions): Promise<object> {
```

Comment:
Why we have replaced interface with object?

Reply:
I have resolved this.